scipy/stats/_continuous_distns.py

```diff
@@ -8854,6 +8854,206 @@ def _updated_ctor_param(self):
         return dct
 
 
+class studentized_range_gen(rv_continuous):
+    """
+    The generalized studentized range continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t: student's t distribution
+
+    Notes
+    -----
+    When `v` is arbitrarily large, in this implementation greater than or equal
+    to 2000, an asymptotic method is used in calculation of the cumulative
+    distribution function. [4]_
+
+    References
+    ----------
+
+    .. [1] "Studentized range distribution",
+           https://en.wikipedia.org/wiki/Studentized_range_distribution
+    .. [2] Batista, Ben Dêivide, et al. "Externally Studentized Normal Midrange
+           Distribution." Ciência e Agrotecnologia, vol. 41, no. 4, 2017, pp.
+           378–389., doi:10.1590/1413-70542017414047716.
+    .. [3] Harter, H. Leon. “Tables of Range and Studentized Range.” The Annals
+           of Mathematical Statistics, vol. 31, no. 4, 1960, pp. 1122–1147.
+           JSTOR, www.jstor.org/stable/2237810. Accessed 18 Feb. 2021.
+    .. [4] Lund, R. E., and J. R. Lund. “Algorithm AS 190: Probabilities and
+           Upper Quantiles for the Studentized Range.” Journal of the Royal
+           Statistical Society. Series C (Applied Statistics), vol. 32, no. 2,
+           1983, pp. 204–210. JSTOR, www.jstor.org/stable/2347300. Accessed 18
+           Feb. 2021.
+
+    Examples
+    --------
+    >>> from scipy.stats import %(name)s
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots(1, 1)
+
+    Calculate a few first moments:
+
+    %(set_vals_stmt)s
+    >>> mean, var, skew, kurt = %(name)s.stats(%(shapes)s, moments='mvsk')
```

Comment:
First time I run this example code, no problem. Second time I get an error. Did we talk about this? Does that happen on your machine @DominicChm?
```
runfile('C:/Users/matth/Dropbox/Scipy/studentized_range/testing.py', wdir='C:/Users/matth/Dropbox/Scipy/studentized_range', current_namespace=True)
Traceback (most recent call last):

  File "C:\Users\matth\Dropbox\Scipy\studentized_range\testing.py", line 16, in <module>
    x = np.linspace(studentized_range.ppf(0.01, k, v),

  File "C:\Users\matth\scipy\scipy\stats\_distn_infrastructure.py", line 2106, in ppf
    place(output, cond, self._ppf(*goodargs) * scale + loc)

  File "C:\Users\matth\scipy\scipy\stats\_distn_infrastructure.py", line 994, in _ppf
    return self._ppfvec(q, *args)

  File "C:\Users\matth\Anaconda3\envs\scipydev\lib\site-packages\numpy\lib\function_base.py", line 2108, in __call__
    return self._vectorize_call(func=func, args=vargs)

  File "C:\Users\matth\Anaconda3\envs\scipydev\lib\site-packages\numpy\lib\function_base.py", line 2192, in _vectorize_call
    outputs = ufunc(*inputs)

  File "C:\Users\matth\scipy\scipy\stats\_distn_infrastructure.py", line 1769, in _ppf_single
    while self._ppf_to_solve(right, q, *args) < 0.:

  File "C:\Users\matth\scipy\scipy\stats\_distn_infrastructure.py", line 1754, in _ppf_to_solve
    return self.cdf(*(x, )+args)-q

  File "C:\Users\matth\scipy\scipy\stats\_distn_infrastructure.py", line 1929, in cdf
    place(output, cond, self._cdf(*goodargs))

  File "C:\Users\matth\scipy\scipy\stats\_continuous_distns.py", line 9050, in _cdf
    return ufunc(x, k, v).astype("float64")

  File "C:\Users\matth\scipy\scipy\stats\_continuous_distns.py", line 9034, in _single_cdf
    opts=dict(epsabs=1e-11, epsrel=1e-12)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 825, in nquad
    return _NQuad(func, ranges, opts, full_output).integrate(*args)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 880, in integrate
    **opt)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 352, in quad
    points)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 465, in _quad
    return _quadpack._qagie(func,bound,infbounds,args,full_output,epsabs,epsrel,limit)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 880, in integrate
    **opt)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 352, in quad
    points)

  File "C:\Users\matth\scipy\scipy\integrate\quadpack.py", line 465, in _quad
    return _quadpack._qagie(func,bound,infbounds,args,full_output,epsabs,epsrel,limit)

ValueError: invalid callable given
```

Reply:
Seems like compilation failed in a way that SciPy found the function but couldn't do anything with it. Has this happened since?